btrfs-property: add page (#7454)
tldr-pages/tldr: pages/linux/btrfs-property.md

# btrfs property

> > Get/Set/List properties for given btrfs filesystem object (file/directory/subvolume/filesystem/device).

Reviewer: Perhaps this will read better?
```suggestion
> Get, set, or list properties for a given btrfs filesystem object (files, directories, subvolumes, filesystems, or devices).
```

> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-property>.

> - List available properties (+description) for the given btrfs object (file/directory/subvolume/filesystem/device):

Reviewer: Perhaps this will read better?
```suggestion
- List available properties (and descriptions) for the given btrfs object:
```

`sudo btrfs property list {{path/to/btrfs_object}}`

- Get all properties for the given btrfs object:

`sudo btrfs property get {{path/to/btrfs_object}}`

> - Get `label` property for the given btrfs filesystem/device:

Reviewer: Perhaps this will read better?
```suggestion
- Get the `label` property for the given btrfs filesystem or device:
```

`sudo btrfs property get {{path/to/btrfs_filesystem}} label`

> - Get all object-type specific properties for the given btrfs filesystem/device:

Reviewer: Perhaps this will read better?
```suggestion
- Get all object type-specific properties for the given btrfs filesystem or device:
```

`sudo btrfs property get -t {{subvol|filesystem|inode|device}} {{path/to/btrfs_filesystem}}`

> - Set `compression` property for the given btrfs inode (file/directory):

Reviewer: Perhaps this will read better?
```suggestion
- Set the `compression` property for a given btrfs inode (either a file or directory):
```
